lib/generators/rolemodel/good_job/templates/config/initializers/good_job.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+Rails.application.configure do
+  config.good_job = {
+    smaller_number_is_higher_priority: true, # new default in V4 and for all of ActiveJob
+    queues: ENV.fetch('GOOD_JOB_QUEUES', 'pdf:1;-pdf'),
```

Comment:
Did you consider adding a comment about the default value? I'm not sure what it does.

Reply:
I agree, this is rather cryptic and should be documented.

Reply:
Yes, I can add a comment.